src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
for e.g. something like this https://github.com/gkz/grasp and then just use it exclusively, no webpack, no babel .... I didn't try it, but worth considering, it might make it much simpler.